### PR TITLE
[ci] Kill hanged docker build process to avoid build timeout issue.

### PR DESCRIPTION
**Pull request by liushilongbuaa**
#### Why I did it
Docker build has a low rate of hanging up.
It hangs on different steps. So, it looks like a bug in docker daemon.
#### How I did it
Start a daemon process to scan running time more than 1 hours, and kill the process.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

**Comment by liushilongbuaa** [comment]
draft

File: .azure-pipelines/cleanup.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 steps:
 - script: |
+    set -x
+    # kill daemon process
```
Comment (xumia) [inline .azure-pipelines/cleanup.yml:4]: Does it cherry pick from master? Looks like missing the logic to check if the file /tmp/azp_daemon_kill_docker_pid exists and the pid exists.

**Review by xumia — APPROVED** [review]

File: .azure-pipelines/cleanup.yml (hunk 1 above)
Comment (liushilongbuaa) [reply .azure-pipelines/cleanup.yml:4]: It is cherry pick from master.
I didn't 'set -e'. If there is not this file. Just go ahead.

**Pull request merged**